src/pvrclient-mythtv.cpp

```diff
@@ -1024,14 +1026,14 @@ PVR_ERROR PVRClientMythTV::GetDeletedRecordings(ADDON_HANDLE handle)
           if (!channel.IsNull())
             strIconPath = m_fileOps->GetChannelIconPath(channel);
         }
-        else
```

Comment:
You have to keep line 1027-1028 to default icon with the preview image, when no cover-art exists.